templates/prescription/general_edit.html

```diff
@@ -277,23 +323,18 @@
     $('#hiddendiv').hide();
 }
 
+function iframetopardiv(string){
+    var name=string
+    document.getElementById('pharmacy').value=name;
+    $("#hiddendiv").html( "&nbsp;" );
+    $('#hiddendiv').hide();
+}
+
 function cancelParlookup () {
     $('#hiddendiv').hide();
     $("#hiddendiv").html( "&nbsp;" );
 }
 
-$().ready(function() {
-    $("#drug").autocomplete('library/ajax/prescription_drugname_lookup.php',
-                            {
-                            width: 200,
-                            scrollHeight: 100,
-                            selectFirst: true
-                            });
-    $("#drug").focus();
-    $("#prescribe").submit(function() { return CheckForErrors(this) });
-    $("#druglookup").click(function() { DoDrugLookup(this) });
```

Comment:
You have removed a lot of functionality here. Doesn't this break for folks that are using current drug look up scheme. Likely need a global to decide which to use (if weno is on, then use the new one).

Also should place 2 autocomplete functions within this $().ready(function() to keep things organized.